### PR TITLE
LOOP-1649: Add illustrations for glucose-related information views

### PR DESCRIPTION
**Pull request by rickpasetto**
- Adds ability to display illustrations in the information views for glucose-related therapy settings
- Includes ability to display a different illustration depending on the unit (mg/dL vs mmol/L)
- For now, only includes mg/dL illustrations

[LOOP-1649](https://tidepool.atlassian.net/browse/LOOP-1649)

### CODE REVIEW

File: LoopKit.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1803,101 +1807,101 @@
 		4369F090208B0D68000E3E45 /* Views */ = {
 			isa = PBXGroup;
 			children = (
-				E9086B3024B3A7270062F5C8 /* ChartTableViewCell.swift */,
```
Comment (darinkrauss) [inline LoopKit.xcodeproj/project.pbxproj:1828]: [nit] What caused all of these deletions and additions?

File: LoopKitUI/Views/Information Screens/InformationScreens.xcassets/glucoseTargetRange mgdL.imageset/Contents.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+  "images" : [
+    {
+      "filename" : "Chart=Correction, Version=V7, Appearance=Light, Unit=mgdL.pdf",
```
Comment (darinkrauss) [inline LoopKitUI/Views/Information Screens/InformationScreens.xcassets/glucoseTargetRange mgdL.imageset/Contents.json:4]: [nit] I prefer to have the name of the file the same as the name of the asset (with suffixes indicated alternations; e.g. "Dark", "Light", etc.). Less confusion.

Comment (darinkrauss) [reply LoopKitUI/Views/Information Screens/InformationScreens.xcassets/glucoseTargetRange mgdL.imageset/Contents.json:4]: Ditto for the rest of the new files.

File: LoopKitUI/Views/Information Screens/GlucoseTherapySettingInformationView.swift [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+//
+
+import HealthKit
+import SwiftUI
+import LoopKit
+
+public struct GlucoseTherapySettingInformationView: View {
+    var text: AnyView!
+    let onExit: (() -> Void)?
+    let mode: PresentationMode
```
Comment (darinkrauss) [inline LoopKitUI/Views/Information Screens/GlucoseTherapySettingInformationView.swift]: [nit] Some confusion between `PresentationMode` type here and `presentationMode` in environment. I understand the intention after reading the code, but it gave me pause at first.

File: LoopKitUI/Views/Information Screens/GlucoseTherapySettingInformationView.swift [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    @Environment(\.presentationMode) var presentationMode
+    
+    public init(
+        therapySetting: TherapySetting,
+        preferredUnit: HKUnit? = nil,
+        text: AnyView? = nil,
+        onExit: (() -> Void)?,
+        mode: PresentationMode = .acceptanceFlow
+    ){
+        self.therapySetting = therapySetting
```
Comment (darinkrauss) [inline LoopKitUI/Views/Information Screens/GlucoseTherapySettingInformationView.swift:29]: [nit] Order of property declarations above, function parameters above, and property assignments here are all different. Consider making the order the same, if possible. It is easier to grok all of the properties together if the order is similar.

File: LoopKit.xcodeproj/project.pbxproj [hunk 5]
```diff
@@ -4036,6 +4113,24 @@
 /* End PBXVariantGroup section */
 
 /* Begin XCBuildConfiguration section */
+		1D81903B25687FF2004D1BEC /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				CODE_SIGN_STYLE = Automatic;
+				DEVELOPMENT_TEAM = 75U4X84TEG;
```
Comment (darinkrauss) [inline LoopKit.xcodeproj/project.pbxproj]: Please remove the `DEVELOPMENT_TEAM` here and below. Thanks!

**Review by darinkrauss — CHANGES_REQUESTED** [review]
Mostly nits, but one change.

File: LoopKit.xcodeproj/project.pbxproj (hunk 1 above)
Comment (rickpasetto) [reply LoopKit.xcodeproj/project.pbxproj:1828]: I sorted the Group in Xcode.  I was hoping we could try to keep the folders/groups sorted to make it easier to navigate.

File: LoopKitUI/Views/Information Screens/InformationScreens.xcassets/glucoseTargetRange mgdL.imageset/Contents.json (hunk 2 above)
Comment (rickpasetto) [reply LoopKitUI/Views/Information Screens/InformationScreens.xcassets/glucoseTargetRange mgdL.imageset/Contents.json:4]: I would prefer to keep the filenames exactly as they are exported in Figma (as shown).  That makes it much easier to update.

File: LoopKitUI/Views/Information Screens/GlucoseTherapySettingInformationView.swift (hunk 3 above)
Comment (rickpasetto) [reply LoopKitUI/Views/Information Screens/GlucoseTherapySettingInformationView.swift]: Ick.  Yeah, that name collision has always been there.  I think I'll rename the former "SettingsPresentationMode"

File: LoopKit.xcodeproj/project.pbxproj (hunk 1 above)
Comment (darinkrauss) [reply LoopKit.xcodeproj/project.pbxproj:1828]: I would suggest that if we are going to do this we do it in a single commit/PR dedicated to resorting all of the folders/groups globally. Doing it piecemeal along with other changes makes it pretty much impossible to see what was actually added/removed. Leave it for now, but please let's do any later sorting separate.

File: LoopKitUI/Views/Information Screens/InformationScreens.xcassets/glucoseTargetRange mgdL.imageset/Contents.json (hunk 2 above)
Comment (darinkrauss) [reply LoopKitUI/Views/Information Screens/InformationScreens.xcassets/glucoseTargetRange mgdL.imageset/Contents.json:4]: Has design settled on this format strategy for all of the Figma files? What if they change their strategy?

**Review by darinkrauss — APPROVED** [review]
LGTM! 🚀

File: LoopKitUI/Views/Information Screens/InformationScreens.xcassets/glucoseTargetRange mgdL.imageset/Contents.json (hunk 2 above)
Comment (rickpasetto) [reply LoopKitUI/Views/Information Screens/InformationScreens.xcassets/glucoseTargetRange mgdL.imageset/Contents.json:4]: I don't think they have a standard strategy for filenames, but there is an "official source" in the Figma file (https://www.figma.com/file/Tf2QUaj26M90tmp4ztS0aY/Therapy-Settings-Acceptance-Flow?node-id=2646%3A851).  As long as I simply export from that source, I should be able to put whatever files here that result from that export, no?

File: LoopKit.xcodeproj/project.pbxproj (hunk 1 above)
Comment (rickpasetto) [reply LoopKit.xcodeproj/project.pbxproj:1828]: Fair enough.  Thanks!

**Pull request merged**